COVID19/src/screens/TestType.js

```diff
@@ -12,14 +12,14 @@ const TestType = ({navigation}) => {
       <Text style={styles.txtTitle}> {constans.testType} </Text>
       <ButtonMain
         captionButton={constans.viralTest}
-        img={require('../utils/images/covidsitos.png')}
+        // img={require('../utils/images/covidsitos.png')}
         onPush={() => {
           navigation.navigate('TestDetails');
         }}
       />
       <ButtonMain
         captionButton={constans.bodyTest}
-        img={require('../utils/images/prueba-de-sangre.png')}
+        // img={require('../utils/images/prueba-de-sangre.png')}
```

Comment:
Try to all the images import and add the import here.

Reply:
fixed